RevenueCatUI/PaywallView.swift

```diff
@@ -104,13 +105,33 @@ public struct PaywallView: View {
         displayCloseButton: Bool = false,
         performPurchase: PerformPurchase? = nil,
         performRestore: PerformRestore? = nil
+    ) {
+        self.init(
+            offering: offering,
+            fonts: fonts,
+            displayCloseButton: displayCloseButton,
+            useDraftPaywall: false,
+            performPurchase: performPurchase,
+            performRestore: performRestore
+            )
+    }
+
+    // swiftlint:disable:next missing_docs
```

Comment:
Idk if we should be documenting these public-but-not-public APIs 🤔  WDYT @RevenueCat/coresdk?

Reply:
I'd be more inclined to documenting since it's quick, and particularly adding a `- warn: this API is meant for internal use`

Reply:
Not sure... I would actually hide them from the reference docs if possible, but maybe we should have a note for people finding it through autocomplete?